packages/eslint-plugin-pf-codemods/lib/rules/v5/table-warn-thExpandType.js

```diff
@@ -0,0 +1,24 @@
+const { getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8634
+module.exports = {
+  meta: {},
+  create: function (context) {
+    const tableImport =  getPackageImports(context, '@patternfly/react-table')
+    .filter(specifier => specifier.imported.name == 'Table' || specifier.imported.name === 'Th');
+
+    return tableImport.length === 0 ? {} : {
+      JSXOpeningElement(node) {
+        if (tableImport.map(imp => imp.local.name).includes(node.name.name)) {
+          const hasCollapseAllAriaLabel = node.attributes.find(n => n.name && n.name.name === 'collapseAllAriaLabel');;
+          if (hasCollapseAllAriaLabel) {
```

Comment:
as we are getting the attribute rather than a boolean, probably name it as such, remove extra `;`

```suggestion
          const collapseAllAriaLabel = node.attributes.find(n => n.name && n.name.name === 'collapseAllAriaLabel');
          if (collapseAllAriaLabel) {
```
bonus might be if we check if it's actually an `as` expression for the value. `collapseAllAriaLabel.?value.?expression?.type` and only apply the warning if it is an `TSAsExpression`

this way the warning wouldn't keep popping up after they fix it and recheck.